<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1133</width>
    <height>689</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="queue1">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>160</y>
         <width>57</width>
         <height>38</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
background-image: url(&quot;D:/QT/program/parkinglot/parkinglot.png&quot;);
font:20px bold;
}</string>
       </property>
       <property name="text">
        <string>1</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="controllabel">
       <property name="geometry">
        <rect>
         <x>600</x>
         <y>80</y>
         <width>101</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
font:25px blod;
}</string>
       </property>
       <property name="text">
        <string>控制台</string>
       </property>
      </widget>
      <widget class="QLabel" name="parkinglot7">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>110</y>
         <width>57</width>
         <height>38</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
background-image: url(&quot;D:/QT/program/parkinglot/parkinglot.png&quot;);
font:20px bold;
}</string>
       </property>
       <property name="text">
        <string>7</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="parkinglot6">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>110</y>
         <width>57</width>
         <height>38</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
background-image: url(&quot;D:/QT/program/parkinglot/parkinglot.png&quot;);
font:20px bold;
}</string>
       </property>
       <property name="text">
        <string>6</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="parkinglot5">
       <property name="geometry">
        <rect>
         <x>390</x>
         <y>70</y>
         <width>57</width>
         <height>38</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
background-image: url(&quot;D:/QT/program/parkinglot/parkinglot.png&quot;);
font:20px bold;
}</string>
       </property>
       <property name="text">
        <string>5</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="parkinglot4">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>70</y>
         <width>57</width>
         <height>38</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
background-image: url(&quot;D:/QT/program/parkinglot/parkinglot.png&quot;);
font:20px bold;
}</string>
       </property>
       <property name="text">
        <string>4</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="parkinglot3">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>70</y>
         <width>57</width>
         <height>38</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
background-image: url(&quot;D:/QT/program/parkinglot/parkinglot.png&quot;);
font:20px bold;
}</string>
       </property>
       <property name="text">
        <string>3</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="parkinglot2">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>70</y>
         <width>57</width>
         <height>38</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
background-image: url(&quot;D:/QT/program/parkinglot/parkinglot.png&quot;);
font:20px bold;
}</string>
       </property>
       <property name="text">
        <string>2</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="parkinglot1">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>70</y>
         <width>57</width>
         <height>38</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
background-image: url(&quot;D:/QT/program/parkinglot/parkinglot.png&quot;);
font:20px bold;
}</string>
       </property>
       <property name="text">
        <string>1</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="queue5">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>320</y>
         <width>57</width>
         <height>38</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
background-image: url(&quot;D:/QT/program/parkinglot/parkinglot.png&quot;);
font:20px bold;
}</string>
       </property>
       <property name="text">
        <string>5</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="queue4">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>280</y>
         <width>57</width>
         <height>38</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
background-image: url(&quot;D:/QT/program/parkinglot/parkinglot.png&quot;);
font:20px bold;
}</string>
       </property>
       <property name="text">
        <string>4</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="queue3">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>240</y>
         <width>57</width>
         <height>38</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
background-image: url(&quot;D:/QT/program/parkinglot/parkinglot.png&quot;);
font:20px bold;
}</string>
       </property>
       <property name="text">
        <string>3</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="queue2">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>200</y>
         <width>57</width>
         <height>38</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
background-image: url(&quot;D:/QT/program/parkinglot/parkinglot.png&quot;);
font:20px bold;
}</string>
       </property>
       <property name="text">
        <string>2</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="parkinglot8">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>110</y>
         <width>57</width>
         <height>38</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
background-image: url(&quot;D:/QT/program/parkinglot/parkinglot.png&quot;);
font:20px bold;
}</string>
       </property>
       <property name="text">
        <string>8</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="parkinglot9">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>110</y>
         <width>57</width>
         <height>38</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
background-image: url(&quot;D:/QT/program/parkinglot/parkinglot.png&quot;);
font:20px bold;
}</string>
       </property>
       <property name="text">
        <string>9</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="parkinglot10">
       <property name="geometry">
        <rect>
         <x>390</x>
         <y>110</y>
         <width>57</width>
         <height>38</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
background-image: url(&quot;D:/QT/program/parkinglot/parkinglot.png&quot;);
font:20px bold;
}</string>
       </property>
       <property name="text">
        <string>10</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="inputlabel">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>390</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
border: 2px solid ；
font:bold italic 16px &quot;微软雅黑&quot;;
}</string>
       </property>
       <property name="text">
        <string> 请输入车牌号</string>
       </property>
      </widget>
      <widget class="QPushButton" name="enterbutton">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>440</y>
         <width>111</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>车辆进入</string>
       </property>
      </widget>
      <widget class="QPushButton" name="outbutton">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>490</y>
         <width>111</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>车辆退出</string>
       </property>
      </widget>
      <widget class="QPushButton" name="checkbutton">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>540</y>
         <width>111</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>查询信息</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="inputnum">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>395</y>
         <width>113</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QScrollArea" name="scrollArea">
       <property name="geometry">
        <rect>
         <x>510</x>
         <y>120</y>
         <width>281</width>
         <height>411</height>
        </rect>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::DefaultContextMenu</enum>
       </property>
       <property name="sizeAdjustPolicy">
        <enum>QAbstractScrollArea::AdjustToContents</enum>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>400</width>
          <height>500000</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>400</width>
          <height>500000</height>
         </size>
        </property>
        <widget class="QLabel" name="showmessage">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>600</width>
           <height>1000000</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>500000</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>-1</pointsize>
           <weight>75</weight>
           <italic>true</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>ArrowCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel{
border-color: rgb(85, 255, 255);
border: 2px solid ;
font:bold italic 16px &quot;微软雅黑&quot;;
}</string>
         </property>
         <property name="text">
          <string>此时停车场为空</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1133</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
